src/Exporter.php

```diff
@@ -337,6 +344,10 @@ private function exportObject(mixed $value, RecursionContext $processed, string
 
         $processed->add($value);
 
+        if ($this->objectExporter !== null && $this->objectExporter->handles($value)) {
+            return $this->objectExporter->export($value);
```

Comment:
not sure but maybe this should be

```suggestion
            return $class . ' Object #' . spl_object_id($value) . ' (' . $this->objectExporter->export($value) . ')';
```

to have a uniform wrapper arround objects